diff --git a/common_func.emevd.js b/common_func.emevd.js
index dfd057c..7281755 100644
--- a/common_func.emevd.js
+++ b/common_func.emevd.js
@@ -9027,11 +9027,7 @@ $Event(90045000, Default, function(eventFlagId, entityId, mapVariationId, logObj
     WaitFor(ActionButtonInArea(9231, entityId) || EventFlag(eventFlagId) || EventFlag(7509));
     EndIf(EventFlag(eventFlagId));
     EndIf(EventFlag(7509));
-    if (!PlayerHasItem(ItemType.Goods, 8000)) {
-        RecordUserDispLog(10108, entityId, LogObjectType.Goods, 8000);
-        WaitFixedTimeSeconds(3);
-        RestartEvent();
-    }
+    // [MOD] Stonesword Key requirement removed: possession check deleted
     if (IsPlayerNo(2)) {
         WaitFor(ElapsedSeconds(0.1));
     }
@@ -9040,7 +9036,7 @@ $Event(90045000, Default, function(eventFlagId, entityId, mapVariationId, logObj
     }
     EndIf(EventFlag(eventFlagId));
     SetNetworkconnectedEventFlagID(eventFlagId, ON);
-    RemoveItemFromPlayer(ItemType.Goods, 8000, 1);
+    // [MOD] Stonesword Key consumption removed
     RecordUserDispLog(11240, 10000, LogObjectType.NpcName, logObjectId);
     SetWorldMapPointFlag(6, ON);
     ForceAnimationPlayback(20000, 60810, false, false, false);
